src/TesApi.Web/Runner/TaskToNodeTaskConverter.cs

```diff
@@ -197,6 +197,14 @@ private static TesOutput PrepareLocalOrLocalCromwellFileOutput(TesOutput output,
                     inputs.AddRange(distinctAdditionalInputs);
                 }
 
+                foreach (var input in inputs)
+                {
+                    if (input?.Path.Contains('?') == true)
+                    {
+                        logger.LogWarning("Warning: TES task with ID {taskId} contains a question mark in its path. The last one will be removed along with all text after it.", task.Id);
+                    }
```

Comment:
Add (inside the braces) `task.SetWarning("Task input path includes '?'. This path was modified. This may break your scripts.");`